sdk/tables/azure-data-tables/samples/sample_create_client.py

```diff
@@ -54,7 +56,7 @@ def create_table_service_client(self):
         # Instantiate a TableServiceClient using a shared access key
         # [START create_table_service_client]
         from azure.data.tables import TableServiceClient
-        with TableServiceClient(account_url=self.account_url, credential=self.access_key) as table_service:
+        with TableServiceClient(account_url=self.account_url, credential=AzureNamedKeyCredential(self.account_name, self.access_key)) as table_service:
```

Comment:
Also for the samples, I think the import statement from `AzureNamedKeyCredential` should be moved into the `[START]` tag so that it shows up in the code snippets - otherwise people might not know where to find it.